.gitlint

```diff
@@ -9,7 +9,7 @@ ignore=B1,B6
 
 # Ensure every title starts with a user-story identifier like US123
 [title-match-regex]
-regex="feat(:|\(.*\):)|fix(:|\(.*\):)|style(:|\(.*\):)|refactor(:|\(.*\):)|test(:|\(.*\):)|docs(:|\(.*\):)|chore(:|\(.*\):)|build(:|\(.*\):)|ci(:|\(.*\):)|perf(:|\(.*\):)|metric(:|\(.*\):)|misc(:|\(.*\):)|release:"gi
+regex=^feat(:|\(.*\):)|fix(:|\(.*\):)|style(:|\(.*\):)|refactor(:|\(.*\):)|test(:|\(.*\):)|docs(:|\(.*\):)|chore(:|\(.*\):)|build(:|\(.*\):)|ci(:|\(.*\):)|perf(:|\(.*\):)|metric(:|\(.*\):)|misc(:|\(.*\):)|release:*
```

Comment:
```suggestion
regex=^feat(:|\(.*\):)|fix(:|\(.*\):)|style(:|\(.*\):)|refactor(:|\(.*\):)|test(:|\(.*\):)|docs(:|\(.*\):)|chore(:|\(.*\):)|build(:|\(.*\):)|ci(:|\(.*\):)|perf(:|\(.*\):)|metric(:|\(.*\):)|misc(:|\(.*\):)|release:
```
The star at the end is wrong I think. It's saying that the `:` after release can occur zero or more times, which allows for instance:
```
release some stuff
```
which I don't think you want to be a valid commit message.

Reply:
I will run some tests. Following the example here: https://jorisroovers.com/gitlint/latest/rules/builtin_rules/#t7-title-match-regex